10-JavaScript/exercises/functions-worksheet.md

```diff
@@ -5,49 +5,122 @@ For Part 2, follow the directions for each problem.
 **Note:** For each function you create, you should also write the code to call the function.
 
 ## Part 1
-1. &nbsp;
-  ```ruby
-  def hello
-    puts 'hello!'
-  end
-  ```
-
-2. &nbsp;
-  ```ruby
-  def say_num(number)
-    puts 'Your number is #{number}.'
-  end
-  ```
-
-3. &nbsp;
-  ```ruby
-  def larger_num(first, second)
-    if first >= second
-      first
-    elsif second > first
-      second
+1. Ruby
+    ```ruby
+    def hello
+      puts 'hello!'
     end
-  end
-  ```
-
-4. &nbsp;
-  ```ruby
-  def output(items)
-    items.each do |item|
-      puts item
+    ```
+
+    <details>
+    <summary>
+    Javascript
+    </summary>
+
+    ```javascript
+    const hello = () => console.log('hello!')
```

Comment:
Because we do arrow functions currently after this lesson, I'm suggesting this change.

```suggestion
    const hello = function() { 
      console.log('hello!');
    }
```